addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1028,6 +1028,90 @@ registry.HeaderHamburgerFull = publicWidget.Widget.extend({
     },
 });
 
+registry.BottomFixedElement = publicWidget.Widget.extend({
```

Comment:
Not a fan of "BottomFixed", this code is generic for elements placed anywhere. I would name the JS and CSS class somthing like "o_overlay_element" if @rdeodoo and you agree

Reply:
It should only apply to bottom element, not anything else IMHO..
If it is somewhere else, you can simply scroll, I don't see a problem.

Reply:
Could be the exact same problem at the top of the page but ok, the code is already hardcoded with bottom anyway. Note that I also did not like the "o_bottom_fixed_element" class as it sounds like a class that *would fix* the element at the bottom... but that's just me :grimacing:

Reply:
we could name the CSS class `o_bottom_overlay_element`
and the js class `BottomOverlayElement`
Tell me what you want, it doesnt matter for me ;)

Reply:
Whatever you want :)